proto/resource_manager.proto

```diff
@@ -38,6 +39,7 @@ message ListResourceGroupsResponse{
 message GetResourceGroupRequest {
   string resource_group_name = 1;
   bool with_ru_stats = 2;
+  uint32 keyspace_id = 3;
```

Comment:
For a cluster that does not enable keyspace, the keyspace id will be u32::MAX after this change, but 0 with the old proto. So in the rolling upgrade scenario, how to distinct between them? Note that 0 is also a valid keyspace id.